src/plugins/intel_npu/src/backend/src/zero_init.cpp

```diff
@@ -147,7 +147,7 @@ ZeroInitStructsHolder::ZeroInitStructsHolder() : log("NPUZeroInitStructsHolder",
 
     // Load our command queue extension
     ze_command_queue_npu_dditable_ext_last_t* _command_queue_npu_dditable_ext = nullptr;
-    if (command_queue_ext_version >= ZE_COMMAND_QUEUE_NPU_EXT_VERSION_1_0) {
+    if (command_queue_ext_version) {
```

Comment:
Does the updated check have the same semantic as `command_queue_ext_version >= ZE_COMMAND_QUEUE_NPU_EXT_VERSION_1_0`?

Reply:
In case it is not supported, the version will be 0. Otherwise, we have a number there, so yes, same thing.